### DISCUSSION TITLE
You are being rate limited. Retry After: 11 seconds

### DISCUSSION THREAD

**Discussion by Sando1285** [RANKTW/Discord-Token-Checker#63] [Q&A]
![photo_2023-10-24_12-28-51](https://github.com/RANKTW/Discord-Token-Checker/assets/72953820/b30e4dcc-1a73-4cf3-876e-8d78246d352a)
Before the update, everything was working fine, after updating, it started giving the following error

**Comment by RANKTW** [comment]
are you using proxy? 
which proxy provider you are using

**Reply by Sando1285** [reply]
I didn’t use proxies and everything worked without a trace

**Reply by RANKTW** [reply]
that's the problem, you have your answer

**Reply by Sando1285** [reply]
Everything was working perfectly until now

**Reply by Sando1285** [reply]
And what kind of proxy is needed

**Reply by RANKTW** [reply]
yes, because Discord has made some updates
you will get the same issue even if you use the older DTC version

**Reply by Sando1285** [reply]
Then suggest which proxies are best to use

**Reply by RANKTW** [reply]
read the [README.md](https://github.com/RANKTW/Discord-Token-Checker#readme)
